kythe/typescript/indexer.ts

```diff
@@ -207,13 +281,9 @@ class StandardIndexerContext implements IndexerHost {
   private rootDirs: string[];
 
   /**
-   * symbolNames maps ts.Symbols to their assigned VNames.
-   * The value is a tuple of the separate TypeScript namespaces, and entries
-   * in it correspond to TSNamespace values.  See the documentation of
-   * TSNamespace.
+   * symbolNames is a store of ts.Symbols to their assigned VNames.
```

Comment:
Single-line this docstring (or remove it entirely).

Reply:
Done.